storagenode/piecestore/endpoint.go

```diff
@@ -164,10 +162,7 @@ func (endpoint *Endpoint) Delete(ctx context.Context, delete *pb.PieceDeleteRequ
 	}
 
 	// TODO: parallelize this and maybe return early
```

Comment:
not sure if this comment should be removed as well

Reply:
Yeah, good call.